integration-tests/plugins/medusa-config.js

```diff
@@ -38,41 +40,95 @@ module.exports = {
     medusa_v2: enableMedusaV2,
   },
   modules: {
-    [Modules.AUTH]: {
-      scope: "internal",
-      resources: "shared",
-      resolve: "@medusajs/auth",
-      options: {
-        providers: [
-          {
-            name: "emailpass",
-            scopes: {
-              admin: {},
-              store: {},
+    workflows: true,
+    ...(enableMedusaV2
+      ? {
```

Comment:
q: whats going on here?

Reply:
removed it, forgot about the modules test suite 😅